backblaze/src/main/java/ch/cyberduck/core/b2/B2UrlProvider.java

```diff
@@ -23,20 +23,20 @@
 import ch.cyberduck.core.Scheme;
 import ch.cyberduck.core.URIEncoder;
 import ch.cyberduck.core.UrlProvider;
+import ch.cyberduck.core.PathRelativizer;
 
 import java.text.MessageFormat;
 import java.util.EnumSet;
 import java.util.Locale;
 
 public class B2UrlProvider implements UrlProvider {
 
-    private final PathContainerService containerService
-        = new B2PathContainerService();
-
+    private final PathContainerService containerService;
     private final B2Session session;
 
     public B2UrlProvider(final B2Session session) {
         this.session = session;
+        this.containerService = session.getFeature(PathContainerService.class);
```

Comment:
Revert this change as the pattern instantiating `B2PathContainerService` is present in all other features.